app/mailers/application_mailer.rb

```diff
@@ -65,11 +65,11 @@ def locale_from_account(account)
   end
```

Comment:
There is a `Current.account` reference in line number 46. I think that also needs to be updated.